okhttp/src/main/java/io/grpc/okhttp/OkHttpProtocolNegotiator.java

```diff
@@ -133,6 +140,59 @@ public String getSelectedProtocol(SSLSocket socket) {
     private static final OptionalMethod<Socket> SET_NPN_PROTOCOLS =
         new OptionalMethod<>(null, "setNpnProtocols", byte[].class);
 
+    // Available on Android 10.0+.
+    // SSLParameters#setApplicationProtocols(String[])
```

Comment:
Since none of these are javadoc, I'd expect it to use the `.` instead of `#`

Reply:
Updated.